Device/Device/doc/spec.md

```diff
@@ -52,8 +52,9 @@ parameters which have to do with the configuration of a device (timeouts, report
 and which are not currently covered by the standard attributes defined by this model. 
     + Attribute type: [StructuredValue](https://schema.org/StructuredValue)
     + Attribute metadata:
-        + `dateModified` :  It captures the last modification timestamp of this attribute.
-            + Type: [DateTime](https://schema.org/DateTime) 
+        + `dateModified` :  Last update timestamp of this attribute.
```

Comment:
this should be changed to `timestamp` to be compliant with the rest of models. an old bug

Reply:
in this case, I was wrong. dateModified is the proper attribute as we are just capturing the last modification date of the device configuration. We can assume that, in this case, the device configuration is changed  directly through this attribute and no delay exists.

Reply:
NTC